features/steps/flaw_detail.py

```diff
@@ -583,25 +584,24 @@ def step_impl(context):
 def step_impl(context, external_system):
     go_to_specific_flaw_detail_page(context.browser)
     flaw_detail_page = FlawDetailPage(context.browser)
-    # Check the tracker jira/bugizlla link
-    flaw_detail_page.click_button_with_js('trackerCount')
-    if external_system == 'jira':
-        flaw_detail_page.trackerJiraherf.visibility_of_element_located()
+    flaw_detail_page.display_affect_detail()
+     # Check the trackers count is not zero
+    flaw_detail_page.trackerCount.visibility_of_element_located()
+    if external_system == 'bugzilla':
+        flaw_detail_page.trackerBzSummary.visibility_of_element_located()
     else:
-        flaw_detail_page.trackerBugzillaherf.visibility_of_element_located()
+        flaw_detail_page.trackerJiraSummary.visibility_of_element_located()
 
 
-@when('When I add a new affect to {external_system} supported module and {affectedness} affectedness')
-def step_imp(context, external_system, affectedness):
+@when('I add a new affect to {external_system} supported module and selected {affectedness_value}')
+def step_imp(context, external_system, affectedness_value):
     go_to_specific_flaw_detail_page(context.browser)
     flaw_detail_page = FlawDetailPage(context.browser)
-    if external_system == 'jira':
-        if affectedness == 'AFFECTED':
-            flaw_detail_page.set_new_affect_inputs(external_system = 'jira', affectedness = 'AFFECTED')
-        if affectedness == 'NEW':
-            flaw_detail_page.set_new_affect_inputs(external_system = 'jira', affectedness = 'NEW')
-    if external_system == 'bugzilla':
-        if affectedness == 'AFFECTED':
-            flaw_detail_page.set_new_affect_inputs(external_system = 'bugzilla', affectedness = 'AFFECTED')
-        if affectedness == 'NEW':
-            flaw_detail_page.set_new_affect_inputs(external_system = 'bugzilla', affectedness = 'NEW')
\ No newline at end of file
+    flaw_detail_page.set_new_affect_inputs(external_system, affectedness_value)
+
+@then("I can't file a tracker")
+def step_impl(context):
+    go_to_specific_flaw_detail_page(context.browser)
+    flaw_detail_page = FlawDetailPage(context.browser)
+    flaw_detail_page.click_button_with_js("ManageTrackers")
+    flaw_detail_page.cannotfileSelectTrackers.visibility_of_element_located()
```

Comment:
Where is the definition of `cannotfileSelectTrackers `, i can not find it.

Reply:
Sorry, I changed it to disabledfileSelectTrackers. Updated.